<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" tests="5" skipped="0" failures="0" errors="0" timestamp="2021-05-01T09:43:17" hostname="wangweilideMBP.lan" time="0.078">
  <properties/>
  <testcase name="negativeTimeout" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.019"/>
  <testcase name="timeoutExceptionThrownIfNoUserException" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.054"/>
  <testcase name="nullNextStatement" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.002"/>
  <testcase name="userExceptionPropagates" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.002"/>
  <testcase name="noExceptionThrownIfNoUserExceptionAndTimeoutDoesNotOccur" classname="org.springframework.test.context.junit4.statements.SpringFailOnTimeoutTests" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
